### PR TITLE
Pull Request H: Michels new fancy method for cancellation effects

### PR DESCRIPTION
**Pull request by jongrumer**
Michels new fancy method for cancellation effects

...

### CODE REVIEW

File: tools/cancellation.f90 [hunk 1]
```diff
@@ -0,0 +1,6 @@
+program cancellation
+      implicit none
+
+      print*, "here we go!"
```
Comment (jongrumer) [inline tools/cancellation.f90:4]: Ok, michel, this looks good - but there must be more to this no? And where is the documentation?!

File: tools/cancellation.f90 [hunk 2]
```diff
@@ -0,0 +1,6 @@
+program cancellation
+      implicit none
```
Comment (jongrumer) [inline tools/cancellation.f90:2]:
```suggestion
      implicit none
      
      integer(kind=4) :: i
```

**Review by minascan — APPROVED** [review]
thats great !

File: tools/cancellation.f90 (hunk 2 above)
Comment (mrgodef) [reply tools/cancellation.f90:2]: Hi Jon! One should be very careful here. Using "CFF" for commenting cards should be strictly avoided after having introded "CF" for "Cancellation Factors"

**Comment by sylaspg** [comment]
Beautiful piece of code! Will be using it, definitely.

**Comment by tspejo** [comment]
Perfect!!
Per

**Pull request closed without merging**